src/chapters/az-cli/intro_az-cli.rst

```diff
@@ -0,0 +1,102 @@
+
+.. admonition:: Fun Fact
+
+    The only interface that exposes greater access to Azure resources over the CLI is the REST API itself. After all, a REST API is just another *interface* that happens to use HTTP requests instead of graphics or the command line. You can make requests directly to the REST API through any HTTP client like a browser, ``curl``, or the ``az CLI`` itself. But most of the time it is a tedious process not meant for day-to-day use when compared to using the GUI or CLI.
+
+There are two key takeaways here about a system, of which Azure is but one example, which supports both a GUI and CLI:
+
+- **the CLI will receive the latest additions and updates before the GUI**
+- **the CLI will have more granular management capabilities that aren't present on the GUI**
+
+While the purpose of this lesson is to inspire your understanding and appreciation for CLIs they *are not always the best choice*. Sometimes GUIs offer an abstraction over more tedious work that can make them worth using. Like most things in the development world you should not blindly adhere to a single approach. What is most important is **to select the right tool for the job** that empowers your workflow. 
+
+Work Velocity
+=============
+
+When it comes to humans interacting with computers there is little doubt that GUIs are more intuitive to work with. But as a technical user your top priority is in choosing an interface that helps you get the job done quickly. On the whole CLIs are the speedier choice for power users because they enable you to issue the exact commands you need and bypass visual distractions. 
+
+CLIs trade ease of exploration, beneficial to newcomers, for brevity and precise control. Whereas GUIs excel in visually guiding you, CLI tools leave it to you to be the guide. They require you to be direct with what you need done but by doing so allow you to complete your tasks more efficiently. 
+
+As an example, let's consider the process of provisioning a new VM from both the web GUI and the ``az CLI``.
+
+Using the web portal you would need to:
+
+#. open your browser
+#. navigate to the azure portal and log in
+#. search for the VM resource
+#. load the "create VM" wizard
+#. work through several menus to configure the VM
+#. confirm and provision
+
+Using the CLI you would need to open your terminal and enter a single command:
+
+.. sourcecode:: bash
+
+    $ az vm create <configuration options>
+
+On the one hand the GUI is helpful in providing visual cues and menus to guide you through the process. On the other hand the CLI allows you to skip directly to issuing the exact instructions for the work you need done. This is just one example that highlights the conciseness with regard to *manual* steps. But the real power of CLI tools comes in their automation capabilities.
+
+Automation Showdown
```

Comment:
We teach about the GUI Delay, and Work Velocity that leads to Automation Showdown, so I think once you've combined the previous 2 sections into 1 this section will stand out more.